File: programmers/level1/숫자문자열과영단어/java/20221028/test/SolutionTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class SolutionTest {

    @Test
    void converter() {
        Solution solution = new Solution();

        assertEquals("1478", solution.convertToStringNumber("one4seveneight"));
    }

    @Test
    void solution() {
        Solution solution = new Solution();

        assertEquals(1478, solution.solution("one4seveneight"));
    }
}

File: programmers/level1/숫자문자열과영단어/javascript/20221028/solution.test.js
import solution, { convertToStringNumber } from './solution';

test('문자열 숫자를 아라비아 숫자로 대체하자', () => {
  expect(convertToStringNumber('one4seveneight')).toBe('1478');
  expect(convertToStringNumber('onetwo39')).toBe('1239');
  expect(convertToStringNumber('onetwoone323')).toBe('121323');
});

test('최종 테스트', () => {
  expect(solution('one4seveneight')).toBe(1478);
  expect(solution('onetwo39')).toBe(1239);
});
